ui/src/main/resources/Meeting/Code/WebHomeSheet.xml

```diff
@@ -36,41 +36,92 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content>{{include reference="Meeting.Code.Migrations.MigrationMacros"/}}
+  <content>{{include reference="Licenses.Code.VelocityMacros"/}}
+{{include reference="Meeting.Code.Migrations.MigrationMacros"/}}
 
 {{velocity}}
-#set($docextras = [])
-#set ($discard = $xwiki.ssfx.use('uicomponents/widgets/userpicker/userPicker.css'))
-#set ($discard = $xwiki.ssfx.use('uicomponents/pagination/pagination.css', 'true'))
-$services.localization.render('contrib.meeting.home.presentation', ['[[**', '**&gt;&gt;Meeting.CalendarView]]'])
-#set ($columnsProperties = {
-  'doc.title': {'type': 'text', 'size': 10, 'link': 'view'},
-  'leader': {'type': 'text', 'size': 10, 'html': true},
-  'place': {'type': 'text', 'size': 10},
-  'startDate': {'type': 'text', 'size': 10},
-  'endDate': {'type': 'hidden'},
-  'durationDisplay': {'type': 'text', 'filterable': true, 'sortable': false, 'html': true},
-  'status': {'type': 'list', 'size': 10},
-  '_actions': {'sortable': false, 'filterable': false, 'html': true, 'actions': ['delete', 'edit']}
-})
-#set ($options = {
-  'className': 'Meeting.Code.MeetingClass',
-  'translationPrefix': 'meeting.livetable.',
-  'tagCloud': true,
-  'rowCount': 15,
-  'selectedColumn': 'startDate'
-})
-#if ($request.getParameter('meeting') == 'all')
-  $services.localization.render('contrib.meeting.home.allMeetingsMessage', ['[[', "&gt;&gt;${doc}]]"])
-  #set ($options.defaultOrder = 'desc')
+#set ($configClassReference = $services.model.createDocumentReference('', ['Meeting', 'Code'], 'ConfigurationClass'))
+#set ($hasLicense = $services.licensing.licensor.hasLicensureForEntity($configClassReference))
+#if (!$hasLicense)
+  {{error}}#getMissingLicenseMessage('meeting.extension.name'){{/error}}
 #else
-  $services.localization.render('contrib.meeting.home.initialMessage', ['[[**', "&gt;&gt;||queryString='meeting=all']]"])
+## Display tabs
```

Comment:
You could move the following code to a Velocity macro, or multiple macros, so that you can indent the code properly.